crypt.go

```diff
@@ -692,6 +744,23 @@ func InitializeLUKS2Container(devicePath, label string, key []byte, options *Ini
 	if len(key) < 32 {
 		return fmt.Errorf("expected a key length of at least 256-bits (got %d)", len(key)*8)
 	}
+
+	// Simplify things a bit
+	// Use a reduced cost for the KDF. This is done because we have a high entropy key rather
+	// than a low entropy passphrase. Setting a higher cost provides no security benefit but
+	// does slow down unlocking. If an adversary is going to attempt to brute force this key,
+	// then they could instead turn their attention to one of the other keys involved in the
+	// protection of this key, some of which can be verified without running a KDF. For
+	// example, with a TPM sealed object, you can verify the parent storage key's seed by
+	// computing the key object's HMAC key and verifying the integrity value on the outer wrapper.
+	defaultKdfOptions := &KDFOptions{TargetDuration: 100 * time.Millisecond}
+	if options == nil {
+		options = &InitializeLUKS2ContainerOptions{KDFOptions: defaultKdfOptions}
+	}
+	if options.KDFOptions == nil {
+		options.KDFOptions = defaultKdfOptions
+	}
+
```

Comment:
In master this is not needed, as luks2.FormatOptions are initialized, adn then InitializeLUKS2COntainerOptions override settings of FormatOptions, which are then used.